### PR TITLE
Add version 9 changes to documentation

### PR DESCRIPTION
**Pull request by oyvindeide**
**Issue**
Resolves #my_issue

**Approach**
_Short description of the approach_

(Screenshot of new behavior in GUI if applicable)

- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [x] Make sure tests pass locally (after every commit!)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

File: docs/getting_started/updating_parameters/fig/release_9.gif
Comment (andreas-el) [inline docs/getting_started/updating_parameters/fig/release_9.gif:1]: We backported a fix for the plotter yesterday, so this graphic might not be a perfect match anymore.

File: docs/about/release_notes.rst [hunk 1]
```diff
@@ -31,6 +27,119 @@ Release Notes
 Highlighted changes
 ===================
 
+Version 9.0
+-----------
+
+New workflow: Evaluate ensemble
+###############################
+
+In this version of ERT we have introduced a new simulation mode called: Evaluate ensemble.
```
Comment (dafeda) [inline docs/about/release_notes.rst:36]:
```suggestion
In this version of ERT we have introduced a new simulation mode called `Evaluate ensemble`.
```

File: docs/about/release_notes.rst [hunk 2]
```diff
@@ -31,6 +27,119 @@ Release Notes
...
+The main reason for this is that in the past, the Ensemble experiment simulation mode has had
```
Comment (dafeda) [inline docs/about/release_notes.rst:37]:
```suggestion
The main reason for this is that in the past, the Ensemble experiment simulation mode has had
```

The primary reason is that the `Ensemble experiment` simulation mode has historically served two distinct purposes: conducting sensitivity analyses and doing manual updates.

File: docs/about/release_notes.rst [hunk 3]
```diff
@@ -31,6 +27,119 @@ Release Notes
...
+two distinct use cases, sensitivity analysis, and after doing a manual update step. But use
```
Comment (dafeda) [inline docs/about/release_notes.rst:38]: "But use cases often ran in an existing experiment and ensemble, but had very different expectations
of what would happen to any parameters already in that experiment.
For the sensitivity analysis
case we would always want any existing parameters to be overwritten, however after doing a manual
update step the parameters should not be resampled."

Is this perhaps a bit easier to follow?

Perhaps the following is 
For instance, running a sensitivity analysis from an existing ensemble required re-sampling and overwriting existing parameters. However, if an `Ensemble experiment` was run after performing a manual update, the parameters should not be resampled.

File: docs/about/release_notes.rst (hunk 1 above)
Comment (dafeda) [inline docs/about/release_notes.rst:36]: I propose leaving out some details. Something like this perhaps:

In this version of ERT, we've introduces a new simulation mode, `Evaluate ensemble`, to address the previously confusing dual functionality of the `Ensemble experiment` simulation mode, which was used both for conducting sensitivity analyses and performing manual updates.
Previously, ERT attempted to automatically discern the user's intention, leading to unpredictable behavior.
To solve this, the Evaluate ensemble mode never overwrites parameters and runs only on ensembles that have parameters but lack responses.
Conversely, Ensemble experiment will now always generate new parameters and create new experiments. 
Each new experiment and ensemble must have a unique name, eliminating the need to delete ERT's internal storage.

File: docs/getting_started/updating_parameters/update_parameters.rst [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+   :scale: 140%
+
+Run forward model
+=================
+
+If we are happy with the draw of the prior, we can run the forward model, and get the responses from the prior. Navigate
+to: `Evaluate ensemble`, and observe that we can see: `Prior` in the `Ensemble` drop down. Press: `Run Experiment` to
+evaluate the prior:
+
+.. image:: fig/release_3.gif
```
Comment (sondreso) [inline docs/getting_started/updating_parameters/update_parameters.rst:34]: The names of these figures are not really intuitive 😅

**Review by sondreso — APPROVED** [review]
Very nice! 📜

**Pull request merged**